libnavigation-metrics/src/main/java/com/mapbox/navigation/metrics/MapboxMetricsReporter.kt

```diff
@@ -98,4 +109,34 @@ object MapboxMetricsReporter : MetricsReporter {
     override fun removeObserver() {
         this.metricsObserver = null
     }
+
+    private class LocationSerializer : JsonSerializer<Location> {
```

Comment:
should add test for this schema?

Reply:
yes, we don't have tests for a final `json`. I'm going to add such tests for `NavigationFeedbackEvent ` in a separate PR.
issue #3381